streams/dz.m3u

```diff
@@ -62,3 +62,5 @@ https://cdn02.hta.dz/abr_htatv/TV8_EDHAKIRA/playlist.m3u8
 #EXTINF:-1 tvg-id="" status="online" user-agent="Mozilla/5.0 (iPhone; CPU iPhone OS 12_2 like Mac OS X) AppleWebKit/605.1.15 (KHTML, like Gecko) Mobile/15E148",AL24 News
 #EXTVLCOPT:http-user-agent=Mozilla/5.0 (iPhone; CPU iPhone OS 12_2 like Mac OS X) AppleWebKit/605.1.15 (KHTML, like Gecko) Mobile/15E148
 https://cdn02.hta.dz/abr_htatv/AL24_News/playlist.m3u8
+#EXTINF:-1 tvg-id="JeelDZTV.dz" status="",Jeel DZ TV (1080p)
```

Comment:
Not 24/7, it may lead you to a 12-hour long stream sometimes.

Reply:
Done.